plugins/catalog-backend/config.d.ts

```diff
@@ -156,6 +156,10 @@ export interface Config {
       | {
           /** Defer stitching to be performed asynchronously */
           mode: 'deferred';
+          /** Polling interval for tasks in seconds */
+          pollingInterval?: number;
```

Comment:
Think it would be better to have these as a `HumanDuration` so you can do something like:

```yaml
pollingInterval: 
  seconds: 30
stitchTimeout: { hours: 2 }
``` 

This aligns with `processingInterval` https://github.com/backstage/backstage/blob/8743beff4d843cc0fe0896a04407369f2db5c902/plugins/catalog-backend/config.d.ts#L184